src/components/EntityInfo.vue

```diff
@@ -57,18 +57,13 @@
                   <span class="font-13 ml-1">View and Change Company Information</span>
                 </v-btn>
 
-                <v-tooltip top content-class="pending-tooltip" v-if="isPendingDissolution || isNotInCompliance">
```

Comment:
Pending Dissolution state will be designed later. Hmm, maybe I will restore this (and text below) and just return False in the getter.